azurerm/resource_arm_container_group.go

```diff
@@ -169,6 +169,13 @@ func resourceArmContainerGroup() *schema.Resource {
 							ForceNew: true,
 						},
 
+						"secure_environment_variables": {
+							Type:      schema.TypeMap,
+							Optional:  true,
+							ForceNew:  true,
+							Sensitive: true,
+						},
```

Comment:
Since this newly added schema, can you also update the documentation for ACI resource?